### PR TITLE
Add APX adapters

### PR DESCRIPTION
**Pull request by AQVYmeCkJsFTFWpMAHmA**
##### Twitter Link:
https://twitter.com/ApolloX_com

##### Website Link:
https://www.apollox.com/en/APX

##### Chain:
BNB Chain

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
apollox-2

##### Short Description (to be shown on DefiLlama):
The world's first centralized and decentralized (CEX-DEX) hybrid crypto exchange that offers trading of Bitcoin, Ethereum and other major cryptocurrencies.

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Derivatives

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/apollox exports TVL: 
        
 
 
    bsc-pool2                 3.21 M
    bsc-staking               453.05 k
    bsc                       14.27 M
    pool2                     3.21 M
    staking                   453.05 k
    
    total                    14.27 M

**Comment by g1nt0ki** [comment]
@AQVYmeCkJsFTFWpMAHmA I have updated the code to make all calls onchain and  split tvl between pool2, staking and rest. If you are fine with the changes I can merge them

**Comment by g1nt0ki** [comment]
@AQVYmeCkJsFTFWpMAHmA sorry, we cant count things that are not on chain

File: projects/apollox/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const { pool2 } = require('../helper/pool2');
+const { staking } = require('../helper/staking');
+const { sumTokens } = require('../helper/unwrapLPs');
+
+const stakingContract = '0x7eb5af418f199ea47494023c3a8b83a210f8846f'
+const stakingContract_APX = '0x6bE863e01E17A226c945e3629D0D9Cb6E52Ce90E'
```
[the remark this answers is not in the document]
Comment (g1nt0ki) [reply projects/apollox/index.js:6]: @AQVYmeCkJsFTFWpMAHmA again, you can not count all the APX and BUSD in LP as your pool2, only the part of LP staked in your contracts, hence I added those two addresses.

Both staking and pool2 can be removed if you desire so, but we cannot use your initial code, it was counting pool2 and staking under tvl, and you were making api calls outside the chain to compute tvl

 for more info: https://docs.llama.fi/list-your-project/submit-a-project

**Pull request merged**